src/components/controls3d/Controls3dDirective.js

```diff
@@ -63,6 +69,17 @@ goog.provide('ga_controls3d_directive');
           }
         };
 
+        scope.resetRotation = function() {
+          var angle = -camera.heading;
+          while (angle < -Math.PI) {
+            angle += Cesium.Math.TWO_PI;
+          }
+          while (angle > Math.PI) {
+            angle -= Cesium.Math.TWO_PI;
+          }
+          scope.rotate(Cesium.Math.toDegrees(angle));
```

Comment:
Is it possible to use the exsting function https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/map/MapService.js#L1443  adding an ol3d param?